docs/source/user_guide_rag.rst

```diff
@@ -547,7 +547,7 @@ The `VectorCypherRetriever` allows full utilization of Neo4j's graph nature by
 enhancing context through graph traversal.
 
 Retrieval Query
------------------------------
+---------------
```

Comment:
This all looks good to me. Reading Ed's feedback however it seems like he's almost after an explanation of how the retriever works. Could be worth updating the first paragraph with a brief description of how it works with references to each of the retriever's arguments. Maybe a diagram also.

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/178/commits/b9fecb2cfd70fa25cda7498c66635f9f00542e77 how's this?